src/JiraCli.Tests/IntegrationTests/IntegrationTests.cs

```diff
@@ -26,15 +27,16 @@ private T ResolveService<T>()
             return ServiceLocator.Default.ResolveType<T>();
         }
 
-        private Context CreateContext(string version)
+        private Context CreateContext(string version, string[] issues = null)
         {
             var context = new Context
             {
                 JiraUrl = JiraUrl,
                 Project = JiraProject,
                 UserName = JiraUser,
                 Password = JiraPassword,
-                Version = version
+                Version = version,
+                Issues = issues
```

Comment:
Maybe best to use this:

```
issues ?? new string[] { }
```

That way we never have to check for null on issues.